boltdesignsystem/bolt
# DS-683 Fix printable element's width in Safari print

## Jira

https://pegadigitalit.atlassian.net/browse/DS-683

## Summary

Fixes a bug where the Printable element does not fit within a page by default in Safari Print.

## Details

Updated CSS to center the printable element to ensure no content is chopped off.

## How to test

Run the branch locally and check the printable doc page (http://localhost:3000/pattern-lab/?p=elements-printable). In Safari, print the page and make sure all content stays within the page. Then check other browsers as well.
